test/parallel/test-cluster-eaccess.js

```diff
@@ -1,41 +1,54 @@
 'use strict';
-// test that errors propagated from cluster children are properly
-// received in their master creates an EADDRINUSE condition by also
-// forking a child process to listen on a socket
-
-var common = require('../common');
-var assert = require('assert');
-var cluster = require('cluster');
-var fork = require('child_process').fork;
-var fs = require('fs');
-var net = require('net');
+// test that errors propagated from cluster workers are properly
+// received in their master. Creates an EADDRINUSE condition by forking
+// a process in child cluster and propagates the error to the master.
 
+const common = require('../common');
+const assert = require('assert');
+const cluster = require('cluster');
+const fork = require('child_process').fork;
+const fs = require('fs');
+const net = require('net');
 
 if (cluster.isMaster) {
-  var worker = cluster.fork();
-  var gotError = 0;
-  worker.on('message', function(err) {
-    gotError++;
+  const worker = cluster.fork();
+
+  // makes sure master is able to fork the worker
+  cluster.on('fork', common.mustCall(function() {}));
+
+  // makes sure the worker is ready
+  worker.on('online', common.mustCall(function() {}));
+
+  worker.on('message', common.mustCall(function(err) {
+    // disconnect first, so that we will not leave zombies
+    worker.disconnect();
+
     console.log(err);
     assert.strictEqual('EADDRINUSE', err.code);
-    worker.disconnect();
-  });
+  }));
+
   process.on('exit', function() {
     console.log('master exited');
     try {
       fs.unlinkSync(common.PIPE);
-    } catch (e) {
+    } catch (ex) {
+      if (ex.code !== 'ENOENT') {
+        throw ex;
+      }
     }
-    assert.equal(gotError, 1);
   });
+
 } else {
-  var cp = fork(common.fixturesDir + '/listen-on-socket-and-exit.js',
+  const cp = fork(common.fixturesDir + '/listen-on-socket-and-exit.js',
                 { stdio: 'inherit' });
```

Comment:
This should be a style error, the arguments don't line up.  I have to admit I'm not a big fan of random style changes, they make the diff a lot noisier (and hence harder to read) than it needs to be.